### PR TITLE
Also output xml when running tests

### PR DESCRIPTION
**Pull request by dstathis**
This would be nice to have for people who want to ingest test result programmatically.

### CODE REVIEW

**Comment by dstathis** [comment]
It would definitely be more convenient if it always outputted the xml and I don't think it would cause too much clutter.

I can easily make it require an environment variable, though, if that's what you decide.

**Comment by Castaglia** [comment]
Yes, please do make it require an environment variable, if you wouldn't mind.  Thanks!

File: tests/api/tests.c [hunk 1]
```diff
@@ -102,7 +102,9 @@ int main(int argc, char *argv[]) {
    * variable or command-line option.
    */
   srunner_set_log(runner, log_file);
-  srunner_set_xml(runner, xml_file);
+  if (getenv("XML_TEST_OUTPUT")) {
```
Comment (Castaglia) [inline tests/api/tests.c]: Thanks!  For consistency, would you object to having this be `PR_XML_TEST_OUTPUT`?

**Comment by Castaglia** [comment]
Thanks!

**Pull request merged**